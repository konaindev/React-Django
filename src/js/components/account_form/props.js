export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
