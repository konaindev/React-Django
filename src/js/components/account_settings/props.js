const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
