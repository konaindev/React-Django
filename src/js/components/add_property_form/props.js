export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
