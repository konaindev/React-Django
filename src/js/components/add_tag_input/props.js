const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
