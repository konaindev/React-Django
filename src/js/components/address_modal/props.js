export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
