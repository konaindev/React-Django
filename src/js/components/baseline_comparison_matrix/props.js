import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
