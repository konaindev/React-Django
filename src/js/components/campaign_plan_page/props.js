import { project, report_links } from "../project_page/props";
import report from "../campaign_plan/campaign_plan.props";

const current_report_link = report_links.market;
const share_info = {
  shared: false,
  share_url: `/projects/${project.public_id}/share/campaign_plan/`,
  update_action: "shared_reports"
};

export default {
  project,
  report,
  report_links,
  current_report_link,
  share_info
};
