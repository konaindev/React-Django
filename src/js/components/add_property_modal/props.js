import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
