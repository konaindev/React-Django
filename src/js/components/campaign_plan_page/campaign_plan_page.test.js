import renderer from "react-test-renderer";

import CampaignPlanPage from "./index";
import props from "./props";
import { Provider } from "react-redux";
import { createStore } from "redux";

describe("CampaignPlanPage", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={createStore(() => props)}>
          <CampaignPlanPage />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
