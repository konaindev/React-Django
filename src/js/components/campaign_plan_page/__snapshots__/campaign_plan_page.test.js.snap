// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampaignPlanPage renders correctly 1`] = `
<div
  className="chrome"
>
  <div
    className="page-chrome__top-chrome"
  >
    <div
      className="page-chrome__top-chrome__content"
    >
      <div
        className="page-header"
      >
        <div
          className="container page-header__inner"
        >
          <a
            href="/dashboard"
          >
            <div
              className="remarkably-logo"
            >
               
            </div>
          </a>
          <nav
            className="page-header__items-outer"
          >
            <ul
              className="page-header__items-inner"
              style={
                Object {
                  "gridTemplateColumns": "repeat(1, 1fr)",
                }
              }
            >
              <li
                className="page-header__item"
              >
                <div
                  className="user-menu"
                >
                  <div
                    className="select user-menu__select css-2b097c-container"
                    onKeyDown={[Function]}
                  >
                    <div
                      className="select__control css-yk16xz-control"
                      onMouseDown={[Function]}
                      onTouchEnd={[Function]}
                    >
                      <div
                        className="select__value-container css-1hwfws3"
                      >
                        <div
                          className="user-menu__image-container"
                        >
                          <div
                            className="user-menu__image user-menu__image--default"
                          />
                          <div
                            className="user-menu__image"
                            style={
                              Object {
                                "backgroundImage": "url(https://storage.googleapis.com/tracker-avatar-production/a11569ffecab2b5564c0cbfa5e5f283a_3140620_96.png)",
                              }
                            }
                          />
                        </div>
                        <input
                          className="css-62g3xt-dummyInput"
                          disabled={false}
                          id="react-select-2-input"
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          readOnly={true}
                          tabIndex="0"
                          value=""
                        />
                      </div>
                      <div
                        className="select__indicators css-1wy0on6"
                      >
                        <span
                          className="select__indicator-separator css-1okebmr-indicatorSeparator"
                        />
                        <div
                          aria-hidden="true"
                          className="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
                          onMouseDown={[Function]}
                          onTouchEnd={[Function]}
                        >
                          <svg
                            aria-hidden="true"
                            className="select__dropdown-arrow select__dropdown-arrow--down"
                            color="inherit"
                            focusable="false"
                            height="1em"
                            role="presentation"
                            viewBox="0 0 10 5"
                            width="1em"
                          >
                            <path
                              d="M1 1.16134L4.92334 4.5"
                              stroke="currentColor"
                              strokeLinecap="round"
                            />
                            <path
                              d="M8.82812 1.16134L5 4.5"
                              stroke="currentColor"
                              strokeLinecap="round"
                            />
                          </svg>
                          <svg
                            aria-hidden="true"
                            className="select__dropdown-arrow select__dropdown-arrow--up"
                            color="inherit"
                            focusable="false"
                            height="1em"
                            role="presentation"
                            viewBox="0 0 10 5"
                            width="1em"
                          >
                            <path
                              d="M1 3.83866L4.92334 0.5"
                              stroke="currentColor"
                              strokeLinecap="round"
                            />
                            <path
                              d="M8.82812 3.83866L5 0.5"
                              stroke="currentColor"
                              strokeLinecap="round"
                            />
                          </svg>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </li>
            </ul>
          </nav>
        </div>
      </div>
      <section
        className="report-page-subheader"
      >
        <div
          className="container"
        >
          <div
            className="report-page__project-link"
          >
            <a
              className="project_link"
              href="/dashboard"
            >
              <div
                className="project_link__arrow"
              />
              <div
                className="project_link__image"
                style={
                  Object {
                    "backgroundImage": "url(\\"https://i.imgur.com/UEH4gfU.jpg\\")",
                  }
                }
              />
              <div
                className="project_link__name"
              >
                Portland Multi Family
              </div>
              <div
                className="property-status property-status--on-track"
              >
                ON TRACK
              </div>
            </a>
          </div>
          <div
            className="subheader__inner"
          >
            <div
              className="project-report-links"
            >
              <ul>
                <li
                  className="enabled"
                  data-url="/projects/pro_example/baseline/"
                  onClick={[Function]}
                >
                  Baseline
                </li>
                <li
                  className="enabled"
                  data-url="/projects/pro_example/market/"
                  onClick={[Function]}
                >
                  Market Analysis
                </li>
                <li
                  className="disabled"
                  data-url="#"
                  onClick={[Function]}
                >
                  Modeling
                </li>
                <li
                  className="selected disabled"
                  data-url="#"
                  onClick={[Function]}
                >
                  Campaign Plan
                </li>
                <li
                  className="enabled"
                  data-url="/projects/pro_example/performance/last-week/"
                  onClick={[Function]}
                >
                  Performance
                </li>
              </ul>
            </div>
            <div
              className="share-toggle"
            >
              <label
                className="button-toggle button-toggle--unchecked"
              >
                <input
                  checked={false}
                  className="button-toggle__input"
                  onChange={[Function]}
                  type="checkbox"
                />
                <span
                  className="button-toggle__track"
                >
                  <span
                    className="button-toggle__thumb"
                  />
                  <div
                    className="button-toggle__checked"
                  >
                    <div
                      className="button-toggle__inner-label"
                    >
                      On
                    </div>
                  </div>
                  <div
                    className="button-toggle__unchecked"
                  >
                    <div
                      className="button-toggle__inner-label"
                    >
                      Off
                    </div>
                  </div>
                </span>
                
              </label>
              <button
                className="button button--disabled button--default copy-to-clipboard share-toggle__copy copy-to-clipboard--disbale"
                onClick={[Function]}
              >
                <svg
                  aria-hidden="true"
                  className="copy-to-clipboard__icon"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 18 8"
                  width="1em"
                >
                  <path
                    d="M4.5 0.25C2.42925 0.25 0.75 1.92925 0.75 4C0.75 6.07075 2.42925 7.75 4.5 7.75H6.75C7.16421 7.75 7.5 7.41421 7.5 7C7.5 6.58579 7.16421 6.25 6.75 6.25H4.5C3.25725 6.25 2.25 5.24275 2.25 4C2.25 2.75725 3.25725 1.75 4.5 1.75H6.75C7.16421 1.75 7.5 1.41421 7.5 1C7.5 0.585786 7.16421 0.25 6.75 0.25H4.5ZM11.25 0.25C10.8358 0.25 10.5 0.585786 10.5 1C10.5 1.41421 10.8358 1.75 11.25 1.75H13.5C14.7428 1.75 15.75 2.75725 15.75 4C15.75 5.24275 14.7428 6.25 13.5 6.25H11.25C10.8358 6.25 10.5 6.58579 10.5 7C10.5 7.41421 10.8358 7.75 11.25 7.75H13.5C15.5708 7.75 17.25 6.07075 17.25 4C17.25 1.92925 15.5708 0.25 13.5 0.25H11.25ZM6 3.25C5.58579 3.25 5.25 3.58579 5.25 4C5.25 4.41421 5.58579 4.75 6 4.75H12C12.4142 4.75 12.75 4.41421 12.75 4C12.75 3.58579 12.4142 3.25 12 3.25H6Z"
                  />
                </svg>
                <span
                  className="copy-to-clipboard__label"
                >
                  Copy Link
                </span>
              </button>
            </div>
          </div>
        </div>
      </section>
    </div>
    <div
      className="page-chrome__top-chrome__spacing-fix"
      style={
        Object {
          "height": 76,
        }
      }
    >
       
    </div>
  </div>
  <div
    className="page campaign_plan-view"
  >
    <div
      className="container"
    >
      <div
        className="campaign_plan-view__subnav"
      >
        <div
          className="button-group"
        >
          <button
            className="button button--selected button--default button-group__item"
            onClick={[Function]}
          >
            Overview
          </button>
          <button
            className="button button--default button-group__item"
            onClick={[Function]}
          >
            Reputation Building
          </button>
          <button
            className="button button--default button-group__item"
            onClick={[Function]}
          >
            Demand Creation
          </button>
          <button
            className="button button--default button-group__item"
            onClick={[Function]}
          >
            Leasing Enablement
          </button>
          <button
            className="button button--default button-group__item"
            onClick={[Function]}
          >
            Marketing Intelligence
          </button>
        </div>
      </div>
      <div
        className="panel panel--default campaign-plan-overview-tab"
      >
        <div
          className="campaign-plan-overview-table"
        >
          <div
            className="table__row"
          >
            <div
              className="row__label"
            >
              Target Segments
            </div>
            <div
              className="target-segments row__content"
            >
              <div
                className="target-segment-item"
              >
                <p>
                  Primary
                </p>
                <p>
                  Young Professionals (25-34)
                </p>
              </div>
              <div
                className="target-segment-item"
              >
                <p>
                  Secondary
                </p>
                <p>
                  Young Entrepreneurs (25-34)
                </p>
              </div>
              <div
                className="target-segment-item"
              >
                <p>
                  Tertiary
                </p>
                <p>
                  Young Couples (35-40)
                </p>
              </div>
              <div
                className="target-segment-item"
              >
                <p>
                  Quarternary
                </p>
                <p>
                  Established Singles (35-44)
                </p>
              </div>
            </div>
          </div>
          <div
            className="table__row"
          >
            <div
              className="row__label"
            >
              Goal
            </div>
            <div
              className="row__content"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "Achieve 95% lease-up by 6/30/2019",
                }
              }
            />
          </div>
          <div
            className="table__row"
          >
            <div
              className="row__label"
            >
              Objectives
            </div>
            <div
              className="objectives-markdown row__content"
            >
              <div
                className="objective-item"
              >
                <h1>
                  Reputation Building
                </h1>
                <div
                  className="markdown-content"
                >
                  <p>
                    Grow reach and shape perceptions
                  </p>
                  <ul>
                    <li>
                      Attract 11,979 USV
                    </li>
                    <li>
                      Attract 1,000 social followers
                    </li>
                    <li>
                      Add 1,100 contacts to property database (email list)
                    </li>
                  </ul>
                </div>
              </div>
              <div
                className="objective-item"
              >
                <h1>
                  Demand Creation
                </h1>
                <div
                  className="markdown-content"
                >
                  <p>
                    Generate INQs and USVs:
                  </p>
                  <ul>
                    <li>
                      Convert 6% of USVs to INQs
                    </li>
                    <li>
                      Attract 696 Inquires (calls, emails, walk-ins, chats, texts)
                    </li>
                    <li>
                      Conduct 270 tours (at 40% INQ&gt;Tour conversion rate)
                    </li>
                  </ul>
                </div>
              </div>
              <div
                className="objective-item"
              >
                <h1>
                  Leasing Enablement
                </h1>
                <div
                  className="markdown-content"
                >
                  <p>
                    Equip leasing team with effective processes and tools that delight
                  </p>
                  <ul>
                    <li>
                      Provide highly responsive process for handling INQ and Tours
                    </li>
                    <li>
                      Capture 97 signed leases
                    </li>
                  </ul>
                </div>
              </div>
              <div
                className="objective-item"
              >
                <h1>
                  Marketing Intelligence
                </h1>
                <div
                  className="markdown-content"
                >
                  <p>
                    Capture resident and prospect data:
                  </p>
                  <ul>
                    <li>
                      Capture prospect data on website
                    </li>
                    <li>
                      Capture resident data about satisfaction and retention
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
          <div
            className="table__row"
          >
            <div
              className="row__label"
            >
              Assumptions
            </div>
            <div
              className="row__content"
            >
              <div
                className="markdown-content"
              >
                <ul>
                  <li>
                    Asset contains 260 units
                  </li>
                  <li>
                    20% C&D rate
                  </li>
                  <li>
                    Model based on predictions generated in Sept. 2018
                  </li>
                </ul>
              </div>
            </div>
          </div>
          <div
            className="table__row"
          >
            <div
              className="row__label"
            >
              Schedule
            </div>
            <div
              className="row__content"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "Campaign begins in Feb. 2019 and finishes in July 2019",
                }
              }
            />
          </div>
          <div
            className="table__row table__row--last"
          >
            <div
              className="row__label"
            >
              Est. Campaign Investment
            </div>
            <div
              className="est-campaign-target row__content"
            >
              <h1>
                $75,500
              </h1>
              <p>
                Based on Current Model
              </p>
              <br />
              <div
                className="target-investments"
              >
                <p>
                  <span>
                    Reputation Building
                  </span>
                  <span />
                  <span>
                    $20,000
                  </span>
                </p>
                <p>
                  <span>
                    Demand Creation
                  </span>
                  <span />
                  <span>
                    $15,000
                  </span>
                </p>
                <p>
                  <span>
                    Leasing Enablement
                  </span>
                  <span />
                  <span>
                    $20,000
                  </span>
                </p>
                <p>
                  <span>
                    Marketing Intelligence
                  </span>
                  <span />
                  <span>
                    $20,500
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
